src/components/list/_list.scss

```diff
@@ -103,15 +106,13 @@ $include-html: false !default;
     @include mint-list-basic-styles();
     display: inline-block;
     vertical-align: inherit;
-    font-size: $breadcrumbListFontSize;
-    line-height: $breadcrumbListFontSize * 1.5;
     color: $breadcrumbListSeparatorColor;
 
     &__element {
       display: inline-block;
 
       &:after {
-        content: '·';
+        content: '•';
```

Comment:
I think that Asia is wrong here - mockups from PS have small dot there. Can we revert that and create an issue to check it with designers? It's not relevant to this change anyway.